whylog/assistant/regex_assistant/regex.py

```diff
@@ -0,0 +1,119 @@
+"""
+Regex verification and creating (but not finding groups in regex)
+"""
+import re
+
+special_characters = frozenset(['.', '^', '$', '*', '+', '{', '}', '?', '[', ']', '|', '(', ')'])
+
+
+class NoDateGroupError(Exception):
```

Comment:
Should be in `whylog/assistant/exceptions.py` and should inherit from `WhylogAssistantError`, which should inherit from `WhylogError`. This way user or developer can catch and handle the exceptions appropriately.